src/OSPSuite.Core/Domain/PKAnalyses/UserDefinedPKParameter.cs

```diff
@@ -111,6 +117,44 @@ public UserDefinedPKParameter()
          return toNullableFloat(estimatedEndTime());
       }
 
-      private float? toNullableFloat(double? estimatedEndTime) => (float?) estimatedEndTime;
+      public double? EstimateDrugMassPerBodyWeight(PKCalculationOptions pkCalculationOptions)
+      {
+         // It can be quite complicated to estimate the total drug mass between two intervals
```

Comment:
@Yuri05 Please review this logic here.
To normalize with DosePerBodyWeight, one need to know the exact drug mass. This becomes quite tricky with Start time and endtime that can be off the start of application

Therefor I am only considering these parameters for tDx and tDy (index of application) WITHOUT any offset